__tests__/app.test.js

```diff
@@ -72,3 +72,42 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe('GET: "/api/reviews/:review_id"', () => {
+  test("200: should get review by correct review id", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body: review }) => {
+        expect(review).toEqual({
```

Comment:
toEqual is too brittle, we want to use objectContaining / matchObject / toHaveProperty to make this test more flexible.